assignment4/count_islands.py

```diff
@@ -0,0 +1,128 @@
+'''
+Created on 23 June 2017
+
+@author: Eva
+'''
+
+from disjoint_set import DisjointSet
+
+def count_islands_iter(rowMax, colMax, tiles):
+    """Counts the number of islands that exist in the given array.
+    
+    This approach uses a disjoint set to merge the land tiles that
+    are adjacent either vertically or horizontally to a given land tile.
+    It then counts the roots of trees that represent land tiles to 
+    calculate the islands.
+    
+    NB. True -> Land tile
+        False -> Water tile
+    NB2. Assume rowMax and colMax are not out of bounds of the array
+        
+        Args:
+            rowMax: an integer, the number of rows in the array
+            colMax: an integer, the number of columns in the array
+            tiles: a two-dimensional array of boolean values
```

Comment:
Instead of the having the NB above, you can say here:
```python
       tiles: a two-dimensional array of boolean values where True represents
               a location with land and False represents a location with water.
```